specification/attributes/attributes.mdpp

```diff
@@ -7,4 +7,5 @@ for data granularity, recency, frequency, etc. Requirements defined in attribute
 
 !INCLUDE "column_naming_convention.md",1
 !INCLUDE "currency_code_format.md",1
+!INCLUDE "datetime_requirements.md",1
```

Comment:
nit: Should we use "format" like the currency code?
```suggestion
!INCLUDE "datetime_format.md",1
```

Reply:
We initially used "format" and then decided to switch to "requirements" due to "represent in UTC requirement". Come to think of it, the "Z" is also part of the format so I can revert to "format"...
Do we need a poll for this? :)

Reply:
I think format is more clear.

Reply:
Switched back to "Date/Time Format", renamed files and adjusted reference to the attribute in Billing Period Start/End specs